test/config/fileserver_config.json

```diff
@@ -0,0 +1,41 @@
+{"os_image":

```

Comment:
@hohene I just make some tricks to avoid use internal repo:
1. I used the server used for OS bootstrapping to store OS iso images, this server is used in PR gate and is public.
2. Test script can fetch the microkernel file directly from RackHD, thus it is not need for a storage space for micro kernel files.